build.sbt

```diff
@@ -1,6 +1,3 @@
-import sbtassembly.AssemblyKeys.assembly
```

Comment:
There were errors when running `sbt show assembly/assemblyOutputPath` until I removed these imports.  The import is automatic already.